tests/DS/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_oscap_test("test_ds.sh")
+add_subdirectory(ds_sds_index)
```

Comment:
quotes

Reply:
@mpreisler Thanks, I'll fix that. The quotes are missing at multiple places, not only here.